### ISSUE TITLE
UnicodeEncodeError with python 2.7 

### ISSUE THREAD

**Issue by vascoosx** [mogproject/mog-commons-python#35]
I am using python 2.7 in cygwin but I am getting  a `UnicodeEncodeError: 'ascii' codec can't encode character` error.

https://github.com/mogproject/mog-commons-python/blob/b0e37fe89d550205fe2876160d6b4c01777a908c/src/mog_commons/command.py#L17

I changed that line to 
```
SHOULD_NOT_ENCODE_ARGS = (six.PY3 and sys.platform == 'win32') or sys.getdefaultencoding() == 'ascii'
```
and now it is working.

It seems like when using python 2.7 in cygwin `SHOULD_NOT_ENCODE_ARGS` should be `true`.

**Comment by mogproject** [comment]
Thank you! I will check it later.

**Comment by mogproject** [comment]
I think I have fixed that in version 0.2.3.
